lib/ohai/config.rb

```diff
@@ -1,6 +1,8 @@
 #
 # Author:: Adam Jacob (<adam@opscode.com>)
+# Author:: Claire McQuin (<claire@chef.io>)
 # Copyright:: Copyright (c) 2008 Opscode, Inc.
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
```

Comment:
I'd check around and make sure this is the right way to do this (two lines), but it seems right offhand ...